common/src/main/webapp/css/react-grid-layout.css

```diff
@@ -64,7 +64,8 @@
 }
 
 .rgl-tile-overlay {
-  display: none; /* Enable for grid debugging */
+  display: none;
```

Comment:
remove debugging

Reply:
It's removed, but prettier doesn't let me put the comment in the same line anymore.